internal/nodes/avi_model_routeingr_hostname_shard.go

```diff
@@ -285,6 +281,7 @@ func ProcessPassthroughHosts(routeIgrObj RouteIngressModel, key string, parsedIn
 			redirect = true
 			hostsMap[host].InsecurePolicy = lib.PolicyRedirect
 		}
+		tenant := lib.GetTenantInNamespace(routeIgrObj.GetNamespace())
```

Comment:
We can take out the tenant derivation out of the loop.

Reply:
Moved out of the loop